fmriprep/workflows/confounds.py

```diff
@@ -243,3 +264,211 @@ def reverse_order(inlist):
     if isinstance(inlist, list):
         inlist.reverse()
     return inlist
+
+
+def get_ica_confounds(ica_out_dir, ignore_aroma_err):
+    import os
+    import shutil
+    import numpy as np
+    from niworkflows.nipype import logging
+
+    # To catch edge cases when there are either no noise or signal components
+    LOGGER = logging.getLogger('workflow')
+
+    # Pass in numpy array and column base name to generate headers
+    # modified from add_header_func
+    def aroma_add_header_func(np_arr, col_base, comp_nums):
+        import pandas as pd
+        from sys import version_info
+        PY3 = version_info[0] > 2
+
+        df = pd.DataFrame(np_arr, columns=[str(col_base) + str(index) for index in comp_nums])
+        df.to_csv(str(col_base) + "AROMAConfounds.tsv",
+                  sep="\t" if PY3 else '\t'.encode(), index=None)
```

Comment:
Okay. We can leave it. I suspect at some point we'll purge `PY3` and similar references, since we don't support Python 2.